### DISCUSSION TITLE
https support

### DISCUSSION THREAD

**Discussion by chowchinho** [mxpv/podsync#275] [Q&A]
first of all, thank you very much on this project and this is helping a lot on my daily listening to a youtube channel

however, I tried to submit the podcast feed to Google Podcast manager, this require it to be https on both the feed and the mp3

any idea how can we change it to be https enabled?

**Comment by alexbergsland** [comment]
Read up on how to setup a reverse proxy.
I use docker for a lot of stuff and I use [Swag from linusserver.io](https://docs.linuxserver.io/general/swag) as web server.
For https you need a certificate, you can get one free from Let's Encrypt or ZeroSSL.
Swag gets and renews them in itself.
Then you need a domain (which I guess you already have), it will then point port 443 to where you have the web server (i.e. Swag) and the reverse proxy will point that to the podsync port.

You can also of course set it up without docker and Swag. With what ever web server you have now.
